sql/share/errmsg-utf8.txt

```diff
@@ -7931,7 +7931,7 @@ ER_VERS_TRUNCATE_VIEW
         eng "DELETE HISTORY from VIEW is prohibited"
 
 ER_VERS_TEMPORARY
-	eng "%s prohibited for TEMPORARY tables"
+	eng "system versioning prohibited for TEMPORARY tables"
```

Comment:
Error messages are capitalized.

Reply:
I'd leave it as it here. Because `system versioning` is not a part of query. It just means `system versioning stuff`. Why it should write `SYSTEM VERSIONING` if one doesn't mention it in query? What do you think?

Reply:
Error messages begin with capitalized letter like this:

System versioning prohibited for TEMPORARY tables

Reply:
Got it. Fixed.